src/Admin/Pool.php

```diff
@@ -40,11 +41,6 @@ class Pool
      */
     protected $adminClasses = [];
 
-    /**
-     * @var string[]
-     */
-    protected $templates = [];
```

Comment:
this is a BC break

Reply:
Can I solve this the same way as in the `AbstractAdmin` class (deprecating the attribute and keep setting it in the `setTemplates()` method)?

Reply:
I would recommend doing so.